expo/gbe/models.py

```diff
@@ -314,21 +313,7 @@ def get_schedule(self):
         Returns schedule as a list of Scheduler.Events
         NOTE:  Things that haven't been booked with start times won't be here.
         '''
-        '''
-        from scheduler.models import Event as sEvent
```

Comment:
This is good - we need to start getting the conference and the scheduler separated out, so this is a good step. (but see above)